src/soilbiogeochem/SoilBiogeochemCarbonFluxType.F90

```diff
@@ -52,6 +56,33 @@ module SoilBiogeochemCarbonFluxType
      real(r8), pointer :: FATES_c_to_litr_cel_c_col                 (:,:)   ! total cellulose    litter coming from ED. gC/m3/s
      real(r8), pointer :: FATES_c_to_litr_lig_c_col                 (:,:)   ! total lignin    litter coming from ED. gC/m3/s
 
+     ! track tradiagonal matrix  
+     real(r8), pointer :: matrix_decomp_fire_k_col                               (:,:)   ! source coefficients
+     real(r8), pointer :: tri_ma_vr                               (:,:)   ! superdiagonal coefficients
+
+
+     type(sparse_matrix_type) :: AKsoilc
```

Comment:
These all need comments to describe what they are.

Reply:
Still needs to be done.

Reply:
done